exdb/tests/integration_tests.py

```diff
@@ -230,6 +230,12 @@ def test_experience_creation_spontaneous_no_conclusion(self):
         form = ExperienceSubmitForm(data, when=self.test_date)
         self.assertFalse(form.is_valid(), "Form should not be valid with no conclusion if it does not need approval")
 
+    def test_experience_creation_invalid_supervisor(self):
+        data = self.get_post_data((self.test_date + timedelta(days=1)), (self.test_date + timedelta(days=2)))
+        data['next_approver'] = self.clients['ra'].user_object.pk
+        form = ExperienceSubmitForm(data, when=self.test_date)
+        self.assertFalse(form.is_valid(), "Form should not be valid if next_approver user does not have 'hs' group")
```

Comment:
It's not really called `hs` anymore. You might just say "Form should not be valid if next_approver user isn't hallstaff".